### PR TITLE
docs(README): add virtual text format tip

### PR DESCRIPTION
**Pull request by jens1205**
Add recommendation about diagnostic config, if diagnostic in neotest is enabled.

### CODE REVIEW

**Comment by akinsho** [comment]
@jens1205 I noticed you removed this from the original PR I don't use the diagnostics feature personally, but I think it's fine to do this formatting automatically inside this plugin. It doesn't seem useful for anyone to have the original with all the bloated stuff you mention. What do you think about just doing it inside the plugin, so people don't have to set up all this stuff themselves? You could make it optional using a config in case for some reason someone complained.

**Comment by jens1205** [comment]
The thing is that if you already remove all tabs, newlines and spaces inside the plugin, the information is lost to early.

I like to have all these formating retained, so I can see it in the diagnostic floating window which pops up when using `vim.diagnostic.goto_next()`. But I don't want to see it in the diagnostic virtual text.

Example how it is working with my suggested setup:
Bildschirmfoto 2022-08-30 um 07 11 42
 
And here a screenshot how it would look like if the tabs / newlines are already stripped in `neotest-go`:

Bildschirmfoto 2022-08-30 um 07 13 39

I prefer the first version.

**Comment by akinsho** [comment]
@jens1205 that's fair enough, the main reason I mentioned was just in case it was possible to make things easier for other people, but as you've explained it's an undesirable trade-off

File: README.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ use({
...
+    		format = function(diagnostic)
+    			local message = diagnostic.message
+    				:gsub("\n", " ")
+    				:gsub("\t", " ")
+    				:gsub("%s+", " ")
+    				:gsub("^%s+", "")
+    			return message
+    		end,
+    	},
+    }, neotest_ns)
```
Comment (akinsho) [inline README.md:30]: @jens1205 is the correct behaviour to create/recreate the neotest namespace or to re-use it? i.e. should the user be doing `api.nvim_get_namespaces()['neotest]` or creating it from scratch. @rcarriga what is the recommended usage here? will it have adverse effects to re-create the neotest namespace?

Comment (akinsho) [reply README.md:30]: Actually, just discovered that `nvim_create_namespace` acts as a get if the name is already in use so doesn't need a separate get here

Comment (rcarriga) [reply README.md:30]: Sorry totally missed this :facepalm: Yes that's correct, it's a weird name :shrug:

**Pull request merged**